js/ttc.js

```diff
@@ -473,13 +483,27 @@ var Controls = (function() {
     	torontoBoundsNE = new google.maps.LatLng(43.930, -79.095),
     	torontoBounds = new google.maps.LatLngBounds(torontoBoundsSW, torontoBoundsNE);
 
+	 // Create a marker for the user's location
+	 window.userloc = new google.maps.Marker({
+		clickable: false,
+		icon: new google.maps.MarkerImage (
+			'//maps.gstatic.com/mapfiles/mobile/mobileimgs2.png',
```

Comment:
How did you find out about this image file? Is there a reference somewhere? (Wondering if there's any other options for styles)

Reply:
From some sample code I found. Unfortunately, Google does not have an official list of their maps icons, and they keep changing the way they are requested (although, at least they seems to keep the legacy methods around too).

From looking at the requests Maps makes via browser Dev Tools, here's an example for the location pin:

https://www.google.com/maps/vt/icon/name=assets/icons/poi/quantum/modern_pinlet/dot_pinlet-2-medium.png&scale=1&highlight=0000FF

The scale param is a real number (e.g. 1.75 works), and highlight is the RGB in hex.